src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -335,4 +335,146 @@ public KeyFrameResponse convertKeyFrame(@RequestParam("asset") MultipartFile ass
+            e.printStackTrace();
+        }
+        if (inputJson.getString("checksum").equals(inputChecksumString)) {
+            new File(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "/").mkdirs();
+            CriteriaThumbnailImage criteria = new CriteriaThumbnailImage();
+            criteria.setSrcPath(properties.getProperty("file.uploadDir") + fileName);
+            criteria.setDestPath(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "/" +
+                    outputJson.getString("name") + outputJson.getString("ext"));
+            ConvertPPTtoPdf converter = new ConvertPPTtoPdf();
+            converter.convert(criteria);
+            String outputChecksumString = "";
+            try {
+                outputChecksumString = checksum.getChecksum(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"), "MD5");
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+            if (configJson.getString("metadata").equals("json")) {
+                //Creation JSON
+                File convertedFile = new File(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"));
+                Metadata metaDataFile = new Metadata();
+                metaDataFile.writeJsonFile(convertedFile);
+            } else {
+                //Creation XMP
+                File convertedFile = new File(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"));
+                Metadata metaDataFile = new Metadata();
+                metaDataFile.writeXmpFile(convertedFile);
+            }
+            if (configJson.getString("thumbnail").equals("True")) {
+                //Creation thumbnail
+                CriteriaPdfToImage criterion = new CriteriaPdfToImage();
+                criterion.setSrcPath(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"));
+                criterion.setDestPath(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "/");
+                criterion.setName(outputJson.getString("name"));
+                criterion.setExt("bmp");
+                criterion.setDpi(150);
+                criterion.setFormatColor("RGB");
+                PdfThumbnail pdf2thumb = new PdfThumbnail();
+                pdf2thumb.convert(criterion);
+            }
+
+            FolderZipped.zipFolder(properties.getProperty("file.downloadDir") + outputJson.getString("name"));
+
```

Comment:
Remove this line.

Reply:
Adopted as this group standard, this line is inserted in order to make readibility easier. At this point we want to stress the difference between the operations carried before the line and after the line.
Conclusion: This blank line will be preserved.